service/gcs/core/gcs/gcs.go

```diff
@@ -472,31 +492,39 @@ func (c *gcsCore) ModifySettings(id string, request prot.ResourceModificationReq
 		return errors.WithStack(gcserr.NewContainerDoesNotExistError(id))
 	}
 
+	settings, ok := request.Settings.(prot.ResourceModificationSettings)
```

Comment:
Yeah, that's true. I agree, though. At least in this point, as long as the errors are produced, the exact order they appear in is less important.